modular-particle-system/src/shapes/rectangle.ts

```diff
@@ -0,0 +1,34 @@
+import { Position } from "../types";
+import { ShapeLogicImplementation } from "./shape";
+
+export interface Rectangle {
+    type: "rectangle";
+    v1: Position;
```

Comment:
I am confused here, what is v1 means? We may give a full name like vertex1

Reply:
Good points. Indeed vertex1 was the source for this. I do agree that meaning `v1` is not obvious.

On another hand, I could appreciate this kind of short property names (v1, v2) in the context of rectangles and triangles, for example, since they are somewhat common. But for people unfamiliar with this kind of programs a full name would definitely be better.

I guess let's rename to `position1`, `position2` and do the same for triangles?